gelweb/gel2mdt/database_utils/multiple_case_adder.py

```diff
@@ -219,7 +219,22 @@ def update_database(self):
             listupdate.reports_added.add(*added_cases)
             listupdate.reports_updated.add(*updated_cases)
 
-            # Add in case alert
+            if self.sample_type == 'raredisease':
+                self.deselect_ensembl_transcripts(added_cases)
+                self.deselect_ensembl_transcripts(updated_cases)
+
+    def deselect_ensembl_transcripts(self, cases):
```

Comment:
Maybe have a docstring here to say why you're doing it and when it will and won't deselect the ensembl transcript

Reply:
Docstring added in